Assets/Dev/Si Wai/Statistics/CurrentStatistic.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using System;
+
+[Serializable]
+public class CurrentStatistic : BaseStatistic
+{
+    [SerializeField]
+    private float baseValue;  // Starting value, read-only
+    private PermanentStatistic permanentStatistic;
+
+    public CurrentStatistic(float bv, PermanentStatistic perm = null) {
+        baseValue = bv;
+        permanentStatistic = perm;
+    }
+
+    // Get the final value after applying modifiers
+    public float GetValue()
+    {
+        float value = baseValue;
+        // first, apply the base multipliers
+        float baseMultiplier = BaseMultipliers();
+        float permanentBaseMultiplier = permanentStatistic?.BaseMultipliers() ?? 1; // if Perm is null, it will be 1
+        value = value * baseMultiplier * permanentBaseMultiplier; // add current * permanent baseMultipliers
```

Comment:
im pretty sure it should not be multiplied, but rather summed 

so base 1.2, permanent 1.3, should become 1.5 total, not 1.6 (1.2 * 1.3) 

unless this was checked w the designers